packages/shared/src/components/profile/ReadingStreaksWidget.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react';
+import { ActivityContainer, ActivitySectionHeader } from './ActivitySection';
+import { ReadingStreakIcon } from '../icons';
+import { useReadingStreak } from '../../hooks/streaks';
+
+interface StreakTagProps {
+  streak: number;
+  title: string;
+  isLoading: boolean;
+}
+const StreakTag = ({ streak, title, isLoading }: StreakTagProps) => {
+  return (
+    <div className="relative flex h-[4.25rem] w-[10.125rem] flex-col rounded-10 border border-background-subtle p-3">
+      {!isLoading && (
+        <>
+          <strong className="typo-title3">{streak}</strong>
+          <p className="text-text-quaternary typo-subhead">{title}</p>
+        </>
+      )}
+    </div>
+  );
+};
+
+export function ReadingStreaksWidget(): ReactElement {
+  const { streak, isLoading } = useReadingStreak();
```

Comment:
You'll see your own streak, even when on other users profile. E.g. I view your profile, it shows my streaks of 494 instead of yours

Reply:
🤦‍♀️  on it